### DISCUSSION TITLE
How to allow only certain notes?

### DISCUSSION THREAD

**Discussion by terheyden** [brianpetro/obsidian-smart-connections#149] [Q&A]
I have a vault with 1100+ notes and I don't want them all indexed by Smart Connections. All my notes live in the same folder.

I have a very specific subset in mind, let's say I want all notes with "Java" in the name to be indexed. How would I do that?

Let's say that's not possible at the moment — if I put all my "Java" notes in a `/java` folder, could I set it up to only index that one folder?

Thanks.

**Comment by brianpetro** [comment]
Hi @terheyden 

Currently, you can exclude folders in the settings.

A setting specific for inclusions would be useful, but currently doesn't exist.
